data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```
@@ -6,58 +6,104 @@
       "_source": {
         "enabled": true
       },
-      "properties": {
-        "droppedEventsCount": {
-          "type": "integer"
+      "dynamic_templates": [
+        {
+          "long_resource_attributes": {
+            "mapping": {
+              "type": "long"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_resource_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "double"
+          }
         },
-        "attributes": {
-          "type": "object"
+        {
+          "long_scope_attributes": {
+            "mapping": {
+              "type": "long"
+            },
+            "path_match": "instrumentationScope.attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_scope_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "instrumentationScope.attributes.*",
+            "match_mapping_type": "double"
+          }
         },
-        "instrumentationScope": {
+        {
+          "long_attributes": {
+            "mapping": {
+              "type": "long"
+            },
+            "path_match": "attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "attributes.*",
+            "match_mapping_type": "double"
+          }
+        }
```

Can you add at least string attributs, if the default was not working? And similar for the other attributes.

```suggestion
        {
          "double_attributes": {
            "mapping": {
              "type": "double"
            },
            "path_match": "attributes.*",
            "match_mapping_type": "double"
          }
        },
        {
          "string_attributes": {
            "mapping": {
              "type": "keyword",
              "ignore_above": 256
            },
            "path_match": "attributes.*",
            "match_mapping_type": "string"
          }
        }

```

---

@KarstenSchnitter I have the change and verified that it works great. Please take a look at the diffs. Let's plan to make any other suggestions you may have in another PR